plonetheme/onegov/upgrades/20190703161348_install_ftw_logo/metadata.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<metadata>
+  <dependencies>
+    <dependency>profile-ftw.logo:default</dependency>
```

Comment:
We don't need both. Either the metadata.xml, or `self.setup_install_profile('profile-ftw.logo:default')` in the upgrade.py. I suggest you delete that one. Now the ftw.logo gets installed twice 😊

Reply:
I strongly recommend to use `ftw.upgrade.step.UpgradeStep#ensure_profile_installed` in order to install Generic Setup Profiles in an Upgrade Step. And never use the `metadata.xml` in the UpgradeStep.

Reply:
👍